apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -208,43 +210,77 @@ export default class ChangeAction extends BaseRushAction {
   /**
    * Asks all questions which are needed to generate changelist for a project.
    */
-  private _askQuestions(packageName: string): Promise<IChangeInfo> {
+  private _askQuestions(packageName: string): Promise<IChangeInfo | undefined> {
     console.log(`${os.EOL}${packageName}`);
-    const bumpOptions: { [type: string]: string } = this._getBumpOptions(packageName);
+    const comments: string[] | undefined = this._changeComments.get(packageName);
+    if (comments && comments.length) {
+      console.log(`Found existing comments:`);
+      comments.forEach(comment => {
+        console.log(`    > ${comment}`);
+      });
+      return this._prompt({
+        name: 'appendComment',
+        type: 'list',
+        default: 'skip',
+        message: 'Append to existing comments or skip:',
```

Comment:
>: [](start = 53, length = 1)

do you need this colon?

Reply:
use a ?

---
In reply to: [147837471](https://github.com/Microsoft/web-build-tools/pull/418#discussion_r147837471) [](ancestors = 147837471)